README.md

```diff
@@ -63,6 +63,7 @@ More configuration parameters:
 - `port`: listen port (default: 24231)
 - `metrics_path`: metrics HTTP endpoint (default: /metrics)
 - `aggregated_metrics_path`: metrics HTTP endpoint (default: /aggregated_metrics)
+- `content_encoding`: encoding format for the exposed metrics (default: identity)
```

Comment:
usually you'd read `Content-Accept` header from the request to determine how to encode the response

Reply:
This is true
But in our case Prometheus server accepts both gzip and identity (it's not specific) so the request header has both in accept encoding. We cannot determine whether we only need compressed or identity. That's why I configured it in server side